src/main/resources/config.yml

```diff
@@ -623,6 +623,16 @@ PlayerCommands:
         Commands:
           Execute: "say §cALERT: §7PLAYER is now a statistician"
           Display: Broadcast
+          
+# When the command /aach add is called for this category.
+Customs:
+  votes:
```

Comment:
`votes` is definitely a good example, several users have requested voting support in the past! 👍